src/main/java/seedu/cafectrl/parser/Parser.java

```diff
@@ -81,8 +81,8 @@ public class Parser {
      * @param pantry The arraylist object created that stores current ingredients in stock
      * @return command requested by the user
      */
-    public static Command parseCommand(Menu menu, String userInput, Ui ui,
-            Pantry pantry, Sales sales, CurrentDate currentDate) {
+    public Command parseCommand(Menu menu, String userInput, Ui ui,
+                                Pantry pantry, Sales sales, CurrentDate currentDate) {
```

Comment:
Shouldn't line 85 be 8 whitespaces from the previous parent line 84?
```suggestion
    public Command parseCommand(Menu menu, String userInput, Ui UI,
            Pantry pantry, Sales sales, CurrentDate currentDate) {
```